### ISSUE TITLE
problem in multi-user setup

### ISSUE THREAD

**Issue by sbecuwe** [broadinstitute/viral-ngs#266]
Hello,

I have installed 1.3.5 using the tar file from the releases page. run_all_tests.sh runs fine.
The directories have 755 permissions such that all users can access all files and run all scripts.

However, if I try to run, say, taxon_filter.py trim_trimmomatic, from the installation directory (followed by all files in my own directory) as a regular user (not the one who has installed it), I get
```
fatal: Not a git repository (or any of the parent directories): .git
```
So I decided to run as the installation user
```
git init
```
in the installation directory. Now I get as a regular user
```
fatal: bad revision 'HEAD'
```
Any idea how to solve it? And what's the best practice the setup viral-ngs in a multi-user environment to avoid that every user should install its own copy?

Thanks!

**Comment by tomkinsc** [comment]
For now, have the user who owns the files install all tools via `nosetests -v test/unit/test_tools.py` (it sounds like you've done this if you have run all tests). Second, have the installing user run one of the top-level Python files, which if running v >1.3.5 will create a text file called `VERSIONS` containing a single line with the current version (you could also create the file manually).

**Comment by sbecuwe** [comment]
Ok, seems to do the job! I thought the VERSIONS problem was fixed in version 1.3.5 since the nosetests were running fine. Thanks!
